pkg/internal/apis/acme/types_order.go

```diff
@@ -70,6 +70,11 @@ type OrderSpec struct {
 	// validation process.
 	// This field must match the corresponding field on the DER encoded CSR.
 	IPAddresses []string
+
+	// Duration is the duration for the not after date for the requested certificate.
+	// this is set on order creation as pe the ACME spec.
+	// +optional
+	Duration *metav1.Duration `json:"duration,omitempty"`
```

Comment:
Ditto. but here you can see that the existing fields do not have json annotations or comment annotations.